### DISCUSSION TITLE
Languages not showing up in forked repository?

### DISCUSSION THREAD

**Discussion by RonaldColyar** [github-linguist/linguist#5260] [Classification] [answered]
Here is my repository: https://github.com/RonaldColyar/music-thing
Here is the original repository: https://github.com/James-Baloyi/music-thing

I have made the last 8 contributions to the original repository through PRs, so I am wondering why are the languages not populated on my repository but are populated on the original? 

Any suggestions/understanding will be appreciated!

**Comment by lildude** [comment, marked as answer]
Normally the language analysis isn't performed when a repo is forked, however it should occur when you next modify a file that contributes to the stats. It looks like something may have interrupted the jobs for your previous commits, but you should find pushing another change to your fork will trigger analysis again.

**Reply by RonaldColyar** [reply]
Made a change to a random JavaScript file and the analysis triggered again. Thanks!
